frontend/src/components/rider/make_delivery/chooseRequestItem.js

```diff
@@ -21,37 +21,43 @@ const ChooseRequestItem = (props) => {
   }
 
   return (  
-     compare() &&  (
-      <div className={styles.container} key={props}>
-    
-        <div className={styles.chooseRequestItem}>        
-          {
-            data.pickupLocationAddress && props.data.pickupLocationAddress.area && (
-              <span className={styles.pickupArea}>
-                <i className="fas fa-map-marker-alt"></i>Pickup:{" "}
-                {props.data.pickupLocationAddress.area}
-              </span>
-            )
-          }
-          <span className={styles.area}>
-            <i className="fas fa-map-marker-alt"></i>Drop:{" "}
-            {data.dropLocationAddress.area}
-          </span>
+     compare() && (  
+        <div className={styles.chooseRequestItem}> 
+          <div className={styles.requesterName}>
+            {props.data.requesterName}
+          </div>       
+          
+          <div className={styles.location}>          
+            {
+              data.pickupLocationAddress && props.data.pickupLocationAddress.area && (
```

Comment:
destructuring and name aliasing makes life easier (a suggestion for future code)